packages/core/src/__test__/integration/two-controlled-props.test.ts

```diff
@@ -0,0 +1,129 @@
+import { createSelector } from '../../selector';
+import { createStore } from '../../store';
+import { StateConfigMap } from 'packages/core/lib/index';
+
+describe('two controlled props (pager like scenario)', () => {
+  const initialState = {
+    pageCount: 10,
+    pageIndex: 10,
+  };
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+    type State = typeof initialState;
+    function createComponent(stateConfig: StateConfigMap<State>) {
+      const store = createStore(initialState, stateConfig);
+      const pageCountVM = jest.fn(({ pageCount }) => `view-${pageCount}`);
+      const pageCountSelector = createSelector(
+        ({ pageCount }: State) => ({ pageCount }),
+        pageCountVM,
+      );
+      const pageIndexVM = jest.fn(({ pageIndex }) => `view-${pageIndex}`);
+      const pageIndexSelector = createSelector(
+        ({ pageIndex }: State) => ({ pageIndex }),
+        pageIndexVM,
+      );
+      store.subscribe((state) => pageCountSelector(state));
+      store.subscribe((state) => pageIndexSelector(state));
+      return {
+        store,
+        pageCountVM,
+        pageIndexVM,
+      };
+    }
+    it('pageCount and pageIndex controlled', () => {
```

Comment:
Let's add a empty line between test cases:

```suggestion
    }
    
    it('pageCount and pageIndex controlled', () => {
```